kubernetes/samples/charts/traefik/README.md

```diff
@@ -57,15 +57,15 @@ middleware.traefik.containo.us/middleware-domain1 created
 ingressroute.traefik.containo.us/traefik-pathrouting-2 created
 middleware.traefik.containo.us/middleware-domain2 created
 ```
-Now you can send requests to different WebLogic domains with the unique Traefik entry point of different paths as defined in route section of `path-routing.yaml`.
+Now you can send requests to different WebLogic domains with the unique Traefik entry point of different paths, as defined in route section of `path-routing.yaml`.
```

Comment:
in the route section
 of the `path-routing.yaml` file.

Reply:
done